internal/app/agent/service.go

```diff
@@ -141,15 +145,13 @@ func (a *agent) Execute(uuid, cmd string) (string, error) {
 	return string(payload), nil
 }
 
-func (a *agent) Control(uuid, cmdStr string) error {
+func (a *agent) Control(uuid, cmdStr string) (err error) {
 	cmdArgs := strings.Split(strings.Replace(cmdStr, " ", "", -1), ",")
 	if len(cmdArgs) < 2 {
 		return errInvalidCommand
 	}
 
-	var resp string
-	var err error
-
+	resp := ""
```

Comment:
If you're using default value initialization, you can use `var resp string` instead of `resp := ""`.

Reply:
i was convinced that his was the prefered way